### PR TITLE
CO-1286 Improve invitations management for calendar events

### PR DESCRIPTION
**Pull request by popod**

### CODE REVIEW

**Comment by ecino** [comment]
Great! Can you add a README file for the module ? You can copy one from another module for the template.

File: calendar_switzerland/README.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Calendar Switzerland
+======================
```
Comment (ecino) [inline calendar_switzerland/README.rst]: The line length should be equal to the title length above

File: calendar_switzerland/README.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Add custom functionality for Switzerland.
```
Comment (ecino) [inline calendar_switzerland/README.rst]: Can you just add a small sentence explaining the email modification you are doing (send invitation e-mails only to employees)

**Review by ecino — CHANGES_REQUESTED** [review]

File: calendar_switzerland/README.rst (hunk 2 above)
Comment (popod) [reply calendar_switzerland/README.rst]: Done.

**Review by ecino — APPROVED** [review]

**Pull request merged**